### DISCUSSION TITLE
How to know the size of the shape?

### DISCUSSION THREAD

**Discussion by Fierce-Ninja** [zsviczian/obsidian-excalidraw-plugin#161] [Q&A] [answered]
Is there a way to know the size of the shape I am drawing?  Something like this (just as a reference), or even on the side bar will do:

![image](https://user-images.githubusercontent.com/12793089/135672996-22efecab-731d-4d36-b37b-c2125b2b3ed1.png)

**Comment by zsviczian** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/14358394/135726682-b5e940e2-352c-4283-bb97-2126f33d674d.png)
